src/gossip/ping_pong.zig

```diff
@@ -129,12 +130,12 @@ pub const PingCache = struct {
         rate_limit_delay_ns: u64,
         cache_capacity: usize,
     ) error{OutOfMemory}!Self {
-        assert(rate_limit_delay_ns <= ttl_ns / 2);
+        std.debug.assert(rate_limit_delay_ns <= ttl_ns / 2);
         return Self{
             .ttl_ns = ttl_ns,
             .rate_limit_delay_ns = rate_limit_delay_ns,
-            .pings = try LruCache(.non_locking, PubkeyAndSocketAddr, Instant).init(allocator, cache_capacity),
-            .pongs = try LruCache(.non_locking, PubkeyAndSocketAddr, Instant).init(allocator, cache_capacity),
+            .pings = try LruCache(.non_locking, PubkeyAndSocketAddr, std.time.Instant).init(allocator, cache_capacity),
+            .pongs = try LruCache(.non_locking, PubkeyAndSocketAddr, std.time.Instant).init(allocator, cache_capacity),
```

Comment:
In contrast but not opposition to my other comment, I think this kind of thing instead warrants extracting the type into a named local declaration somewhere to avoid this kind of duplication, ie:
```zig
const Pings = LruCache(.non_locking, PubkeyAndSocketAddr, std.time.Instant);
const Pongs = LruCache(.non_locking, PubkeyAndSocketAddr, std.time.Instant);
```
maybe only one type like `PingsOrPongs` if the type of these fields are "intertwined" (as in, probably never going to differ).